compiler/control/OMROptions.cpp

```diff
@@ -2590,6 +2591,7 @@ OMR::Options::jitPreProcess()
    self()->setOption(TR_EnableCodeCacheConsolidation);
 #endif
 
+   self()->setOption(TR_DisableZNext);
```

Comment:
Are we planning to reenable this in the future?

Reply:
The intention is to disable by default currently -- enabled only with -Xjit:enableZNext.  We will flip the switch in the future.